### DISCUSSION TITLE
Path styles import is not working

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2638] [Q&A] [answered]
Here is the example with button.
https://codesandbox.io/s/lucid-volhard-dgrb2u?file=/src/index.js

**Comment by simonguo** [comment, marked as answer]
The codesandbox example you provided may not have been saved, I don't see the problem you are trying to express.

**Reply by abdullaev388** [reply]
yes sorry, here is it
https://codesandbox.io/s/beautiful-johnson-9ic51j?file=/src/index.js

**Reply by simonguo** [reply]
There is an issue with importing less styles on demand, we will fix it soon.

**Reply by abdullaev388** [reply]
> There is an issue with importing less styles on demand, we will fix it soon.

will wait!

**Reply by abdullaev388** [reply]
is it fixed?

**Reply by simonguo** [reply]
As far as the example in codesandbox is concerned, there should be a problem with parsing less inside codesandbox. Because in the example we provided there is no problem.  You can try to run the following example:
https://github.com/rsuite/rsuite/tree/main/examples/create-react-app

At present, the problem we have found is that the on-demand loading fails to match the corresponding style file dependencies. For example: importing DatePicker requires importing the style files it depends on.

```js
import DatePicker from 'rsuite/DatePicker';
import 'rsuite/Calendar/styles/index.less';
import 'rsuite/Picker/styles/index.less';
import 'rsuite/DatePicker/styles/index.less’;
```
We're simplifying this usage, but it won't be all fixed anytime soon, so please be patient.
